projects/from-javascript-to-typescript/the-typeinator/jest.config.js

```diff
@@ -1,5 +1,12 @@
 module.exports = {
-	fakeTimers: { enableGlobally: true },
```

Comment:
Looking at https://github.com/LearningTypeScript/projects/runs/6781807752?check_suite_focus=true, typinator projects are failing. I think it's from removing this line?

```
    console.warn
      A function to advance timers was called but the timers APIs are not replaced with fake timers. Call `jest.useFakeTimers()` in this test file or enable fake timers for all tests by setting 'fakeTimers': {'enableGlobally': true} in Jest configuration file.
      Stack Trace:
            4 |
            5 | const simulateSetTimeoutResolve = async (time) => {
          > 6 | 	jest.advanceTimersByTime(time);
              | 	     ^
            7 |
            8 | 	// Wait for a seemingly larger than needed number of resolves,
            9 | 	// just in case the code does more async/await than it needs...
            Error: 
            at FakeTimers._checkFakeTimers (../../../node_modules/@jest/fake-timers/build/modernFakeTimers.js:152:13)
            at FakeTimers.advanceTimersByTime (../../../node_modules/@jest/fake-timers/build/modernFakeTimers.js:85:14)
            at advanceTimersByTime (02-callbacks-to-async-await/index.test.js:6:7)
            at Object.simulateSetTimeoutResolve (02-callbacks-to-async-await/index.test.js:23:9)
      4 |
      5 | const simulateSetTimeoutResolve = async (time) => {
    > 6 | 	jest.advanceTimersByTime(time);
        | 	     ^
      7 |
      8 | 	// Wait for a seemingly larger than needed number of resolves,
      9 | 	// just in case the code does more async/await than it needs...
      at FakeTimers._checkFakeTimers (../../../node_modules/@jest/fake-timers/build/modernFakeTimers.js:146:28)
      at FakeTimers.advanceTimersByTime (../../../node_modules/@jest/fake-timers/build/modernFakeTimers.js:85:14)
      at advanceTimersByTime (02-callbacks-to-async-await/index.test.js:6:7)
      at Object.simulateSetTimeoutResolve (02-callbacks-to-async-await/index.test.js:23:9)
```

Reply:
Omg. I didn't notice, becuase I wrote a bash script to update all jest.config.js. I have made changes.